src/components/TimeSlider/styles.ts

```diff
@@ -0,0 +1,128 @@
+import { GrafanaTheme2 } from '@grafana/data';
+import { css } from '@emotion/css';
+import { css as cssCore } from '@emotion/react';
+
+export const getStyles = (theme: GrafanaTheme2, hasMarks = false) => {
+  const railColor = theme.colors.border.strong;
+  const trackColor = theme.colors.primary.main;
+  const handleColor = theme.colors.primary.main;
+  const blueOpacity = theme.colors.primary.transparent;
+  const hoverStyle = `box-shadow: 0px 0px 0px 6px ${blueOpacity}`;
+
+  return {
+    container: css({
+      width: '100%',
+      margin: 'inherit',
+      paddingBottom: hasMarks ? theme.spacing(1) : 'inherit',
+      height: 'auto',
+    }),
+    // can't write this as an object since it needs to overwrite rc-slider styles
+    // object syntax doesn't support kebab case keys
+    slider: css`
+      .rc-slider {
```

Comment:
I just checked by doing and this worked for me:

```
slider: css({
      ['.rc-slider']: {
        display: `flex`,
        flexGrow: 1,
        marginLeft: `7px`,
      },
    }),
  
```

Reply:
Yeah, I figured not worth the toil since it's just a direct copy of what's in @grafana/ui, and I'm assuming we'll have to update the styles to match at some point. I'll update to the object syntax